reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py

```diff
@@ -72,6 +72,13 @@ def pr_is_blocked(env_vars: dict) -> bool:
     config = get_approved_files_config(repo)
     approved_files = get_approved_files(config)
     block_pr = not all(file in approved_files for file in changed_files)
+    if block_pr:
+        print(
+            f"""Blocking PR because the changed files are not in the list of approved files.
+                Update config at: {BOT_APPROVED_FILES_PATH} if necessary.
```

Comment:
This error message is shown on a PR right? Maybe also include a link to this repo besides the `BOT_APPROVED_FILES_PATH` so folks can find it.

Reply:
Well the config file would be located in the individual repo, not this repo. I don't really foresee cases where they'd need to change anything to the workflow logic. I've added a link to this repo and workflow in the README but I don't really think it's needed for the error message. WDYT?

Reply:
Oh yes I didn't realize that. All good then!